extensions/amp-analytics/0.1/requests.js

```diff
@@ -36,8 +36,16 @@ export class RequestHandler {
    * @param {!../../../src/preconnect.Preconnect} preconnect
    * @param {./transport.Transport} transport
    * @param {boolean} isSandbox
+   * @param {string} requestOrigin
    */
-  constructor(element, request, preconnect, transport, isSandbox) {
+  constructor(
+    element,
+    request,
+    preconnect,
+    transport,
+    isSandbox,
+    requestOrigin
```

Comment:
Given we may allow each request object to override requestOrigin in the future, I think it will look much cleaner if we can write the `requestOrigin` value intro each individual `request` object instead of passing the argument into the constructor.

Reply:
Also there's the tradeoff between resolving the `requestOrigin` multiple times, versus keep the code clean. 
I prefer the previous approach because here resolving macro doesn't fall on the critical path, also because we may need to support different `requestOrigin` for different request. Let me know what do you think.

Reply:
I agree, fixed in latest commit